pom.xml

```diff
@@ -35,6 +35,10 @@
 			<artifactId>hamcrest-all</artifactId>
 			<version>1.3</version>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+		</dependency>
 		<!--  Java SemVer v0.9.0 (SemVer 2) -->
```

Comment:
I think this dependency should be going as part of RPE-18 but happy for it to be covered by a future PR if this is just a first step in that user story